indexer/feature_meta.hpp

```diff
@@ -126,6 +126,10 @@ class Metadata : public MetadataBase
     FMD_SPONSORED_ID = 24,
     FMD_PRICE_RATE = 25,
     FMD_RATING = 26,
+    FMD_SPONSORED_VENDOR = 27,
```

Comment:
I'm worried that the number of metadata fields grows soo fast. Could you please estimate how many features per mwm will have these fields? Why not to create a separate section for sponsored objects?